media-sound/fluid-soundfont/fluid-soundfont-3.1.ebuild

```diff
@@ -1,29 +1,62 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-DESCRIPTION="The Fluid R3 soundfont"
-HOMEPAGE="http://musescore.org/en/handbook/soundfont"
+# There are no new releases for old data files but the archival repo can update
+# the accompanying documentation [but this ebuild only cares for the data ;) ].
+COMMIT=700c7c1775da12f630750cc34149763ee52aa6c6
+
+DESCRIPTION="The Fluid R3 version 1 soundfont"
+HOMEPAGE="https://github.com/pinkflames/fluid-soundfont"
 SRC_URI="
-	http://prereleases.musescore.org/soundfont/${PN}_${PV}.tar.gz
-	timidity? ( https://dev.gentoo.org/~hwoarang/distfiles/timidity.cfg.bz2 )"
+	https://github.com/pinkflames/${PN}/archive/${COMMIT}.tar.gz -> ${PN}-${COMMIT}.tar.gz
+"
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~ppc64 x86"
-IUSE="timidity"
+IUSE="+gm gs"
+
+BDEPEND="media-libs/flac"
+
+REQUIRED_USE="|| ( gm gs )"
+
+DOCS=( original/COPYING original/README )
+
+S="${WORKDIR}"/${PN}-${COMMIT}
+
+src_prepare() {
+	einfo "Using the flac tool to decompress the soundfont(s):"
+	use gm && flac --force-raw-format --endian=little --sign=signed -d original/FluidR3_GM.sf2.flacy -o FluidR3_GM.sf2
+	use gs && flac --force-raw-format --endian=little --sign=signed -d original/FluidR3_GS.sf2.flacy -o FluidR3_GS.sf2
 
-RDEPEND="timidity? ( app-eselect/eselect-timidity )"
+	default
+}
 
 src_install() {
-	insinto /usr/share/sounds/sf2
-	doins FluidR3_GM.sf2 FluidR3_GS.sf2
+	default
 
-	dodoc README
+	insinto /usr/share/sounds/sf2
+	use gm && doins FluidR3_GM.sf2
+	use gs && doins FluidR3_GS.sf2
 
-	if use timidity; then
+	if use gm; then
+		# Since  enabling is done via eselect timidity, it's safe to always install our patchset
 		insinto /usr/share/timidity/${PN}
-		doins "${WORKDIR}"/timidity.cfg
+		doins contrib/timidity.cfg
+	fi
+}
+
+pkg_postinst() {
+	elog "We're not lawyers but, according to Debian, music produced from samples"
+	elog "under MIT licence may itself be subject to it, if the samples constitute"
+	elog "a substantial portion of the work. In particular this means reproducing"
+	elog "the copyright notice and the licence when distributing derived works."
```

Comment:
```suggestion
	elog "Music produced from samples under MIT licence may itself be subject to"
	elog "it, if the samples constitute a substantial portion of the work. In" 
	elog "particular this means reproducing the copyright notice and the licence"
	elog "when distributing derived works. This is not legal advice."
```

Reply:
Besides I heard an opinion that this should be printed once, so you should utilize *readme-gentoo-r1.eclass*.

Reply:
On one hand, yes, this might be mundane enough to print it only once, so I have added it to the TODO for later but in general I do not believe that the issue is packages print too much elog messages - some people do not read them even when doing the first installation of one particular package and having been told to read and and follow instructions that the particular ebuild will print.